runtime/executor/src/main/java/org/apache/nemo/runtime/executor/datatransfer/OperatorVertexOutputCollector.java

```diff
@@ -42,9 +42,13 @@
   private static final Logger LOG = LoggerFactory.getLogger(OperatorVertexOutputCollector.class.getName());
 
   private final IRVertex irVertex;
+  @SuppressWarnings("squid:S1948")
   private final List<NextIntraTaskOperatorInfo> internalMainOutputs;
+  @SuppressWarnings("squid:S1948")
   private final Map<String, List<NextIntraTaskOperatorInfo>> internalAdditionalOutputs;
+  @SuppressWarnings("squid:S1948")
   private final List<OutputWriter> externalMainOutputs;
+  @SuppressWarnings("squid:S1948")
```

Comment:
Please remove the four `@SuppressWarnings` here. `@SuppressWarnings` is reasonable, yet let's keep this part untouched so that we handle this warning later.